test/e2e/kubevirt.go

```diff
@@ -393,9 +407,10 @@ passwd:
 
 			Expect(fr.ClientSet.NetworkingV1().NetworkPolicies(namespace).Delete(context.TODO(), policy.Name, metav1.DeleteOptions{})).To(Succeed())
 
-			// Wait some time for network policy removal to take effect
-			time.Sleep(1 * time.Second)
+			Expect(reconnect(endpoints)).To(Succeed(), step)
```

Comment:
shouldn't this be in an eventually with a retry? Otherwise you would be assuming that we react to the deleted policy instantly.

Reply:
We are already doing a `retry.OnError` at the `dial` function.